src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -723,8 +730,20 @@ export const selectIsAnyTradeQuoteLoading = createSelector(
 )
 
 export const selectIsAnyTradeQuoteLoaded = createSelector(
-  selectIsSwapperQuoteAvailable,
   selectHasUserEnteredAmount,
-  (isSwapperQuoteAvailable, hasUserEnteredAmount) =>
-    !hasUserEnteredAmount || Object.values(isSwapperQuoteAvailable).some(identity),
+  selectIsAnyTradeQuoteLoading,
+  selectIsAnySwapperQuoteAvailable,
+  (hasUserEnteredAmount, isAnyTradeQuoteLoading, isAnySwapperQuoteAvailable) => {
+    if (!hasUserEnteredAmount) {
+      return true
+    }
+
+    // If we're still loading, return true if there is any quote available
+    if (isAnyTradeQuoteLoading) {
+      return isAnySwapperQuoteAvailable
+    }
```

Comment:
q: does it matter or not whether any is loading? High-level feels like `isAnySwapperQuoteAvailable` should be all we need

Reply:
The selector naming is off here, per Apotheosis's comment. Really this selector is driving whether to treat the quotes as "loading" from a user perspective more than anything else.

Reply:
here we go, went with selectShouldShowTradeQuoteOrAwaitInput which is hopefully a better representation of what's happening https://github.com/shapeshift/web/commit/c1fa8be3b9c23b89ea65cdb168367f70332b418a